libs/features/personalization/personalization.js

```diff
@@ -680,13 +665,37 @@ export async function runPersonalization(experiment, config) {
   };
 }
 
+function overridePersonalizationVariant(manifest, config) {
+  const { manifestPath, variantNames } = manifest;
+  if (!config.mep?.override) return;
+  let selectedVariant;
+  config.mep?.override?.split('---').some((item) => {
+    const pair = item.trim().split('--');
+    if (pair[0] === manifestPath && pair.length > 1) {
+      [, selectedVariant] = pair;
+      return true;
+    }
+    return false;
+  });
+  if (selectedVariant) {
+    if (variantNames.includes(selectedVariant)) {
+      manifest.selectedVariantName = selectedVariant;
+      manifest.selectedVariant = manifest.variants[selectedVariant];
+    } else {
+      manifest.selectedVariantName = selectedVariant;
+      manifest.selectedVariant = manifest.variants[selectedVariant];
+    }
+  }
```

Comment:
I'd avoid the nested if elses
```suggestion
  if (!selectedVariant) return
  if (variantNames.includes(selectedVariant)) {
    manifest.selectedVariantName = selectedVariant;
    manifest.selectedVariant = manifest.variants[selectedVariant];
    return
  } 
  manifest.selectedVariantName = selectedVariant;
  manifest.selectedVariant = manifest.variants[selectedVariant];
```